docs/how-to/visualize_views.md

```diff
@@ -0,0 +1,37 @@
+# How-To: Visualize Views
+
+To create simple UI interface use [GradioAdapter class](../../src/dbally/gradio/gradio_interface.py) It allows to display Data Preview related to Views
+and execute user queries.
+
+## Installation
+```bash
+pip install dbally[gradio]
+```
+
+## Create own gradio interface
+Define collection with implemented views
+
+```python
+    llm = LiteLLM(model_name="gpt-3.5-turbo")
+    collection = dbally.create_collection("recruitment", llm, event_handlers=[CLIEventHandler()])
+    collection.add(CandidateView, lambda: CandidateView(engine))
+    collection.add(SampleText2SQLViewCyphers, lambda: SampleText2SQLViewCyphers(create_freeform_memory_engine()))
+```
+
+Create gradio interface
+```python
+    gradio_interface = await create_gradio_interface(user_collection=collection)
+    if gradio_interface:
+        gradio_interface.launch()
+```
+
+Launch the gradio interface. To publish public interface pass argument `share=True`
+```python
+    gradio_interface.launch()
+```
+
+The endpoint is set by by triggering python module with Gradio Adapter launch command.
+Private endpoint is set to http://127.0.0.1:7860/ by default.
+
+## Links
+* [Example Gradio Interface](../../examples/visualize_views_code.py)
```

Comment:
This link doesn't work, please replace it with a link to the file in github repo.